lib/strategies/webapp-strategy.js

```diff
@@ -528,4 +530,28 @@ function logAction(req, url, activity) {
 	});
 }
 
+/**
+ * This method will trigger a REST HTTP call to the AppID server logoutSSO endpoint.
+ * Bare in mind that SSO feature must be activated in the server side.
+ * Calling the server SSO APIs will do nothing if you forgot to enable SSO in the server (tenant) configuration.
+ * Once you enabled SSO login in the server side you can try the logout in the client side:
+ * If you use the sample application , add a logout SSO UI widget , add to the app.js an express routing such as
+ * app.get("/logoutSSO", function(req, res, next) {
+ *	res.clearCookie("refreshToken");
+ *	webAppStrategy.logoutSSO(req,res, { successRedirect: "http://localhost:3000/niceGoodbyePage" });
+ * });
+ * @param req the http request
+ * @param res - the response object (will be used to redirect)
+ * @param options - need to contain successRedirect.
+ */
+WebAppStrategy.prototype.logoutSSO = function (req, res, options = {}) {
+	let redirectUri = options.successRedirect || '';
```

Comment:
I think we should not use the term "successRedirect" as it's very confusing. (in other places we use it as just the suffix without the context and here we should pass the full url)

Reply:
ok. changed the option JSON to be passed AS IS